injection/interface.go

```diff
@@ -53,9 +53,16 @@ type Interface interface {
 	// a new informer with a context.
 	RegisterInformer(InformerInjector)
 
+	// RegisterInformers registers a new filtered informer injector callback for associating
+	// a new set of informers with a context.
+	RegisterFilteredInformer(FilteredInformersInjector)
```

Comment:
should this method be plural to match the argument it takes?

```suggestion
	RegisterFilteredInformers(FilteredInformersInjector)
```

Reply:
..though having said that, I'm wondering if we actually want FilteredInformersInjector to be plural anyway. Could it match RegisterInformer(InformerInjector) and be singular (and inject a single filteredinformer at a time), or is there a reason it has to differ?

Reply:
@julz Thank you for your comments. The plural informers comes from this comment: https://github.com/knative/pkg/pull/1940#discussion_r541150349
And pair work with @mattmoor to have this design.